src/Tools/DynamoFeatureFlags/FeatureFlagsClient.cs

```diff
@@ -137,16 +137,10 @@ private static string GetConfigurationItem(String key,string configFilePath)
         {
             try
             {
-                var doc = new XmlDocument();
-                doc.Load(configFilePath);
-                if (doc != null)
-                {
-                    XmlNode node = doc.SelectSingleNode("//appSettings");
-
-                    XmlElement value = (XmlElement)node.SelectSingleNode(string.Format("//add[@key='{0}']", key));
-                    return value.Attributes["value"].Value;
-                }
-                return null;
+                IConfigurationRoot config = new ConfigurationBuilder()
```

Comment:
do we want to have this code cross target framework ?

Reply:
This bussines logic will be ported in the future, so we don't break any test